### PR TITLE
Add chart testing

### LINKED ISSUE #11: Enable helm chart test
It's better to have some automation tests which has been already written.
I don't know what is your CI service, but we can do it using github actions or something else
https://github.com/helm/chart-testing

### PR DESCRIPTION
**Pull request by Jasstkn**
Signed-off-by: Jasstkn <jasssstkn@yahoo.com>

closes #11

### CODE REVIEW

**Comment by Jasstkn** [comment]
Screenshot 2020-05-01 at 13 39 43

**Comment by Jasstkn** [comment]
Screenshot 2020-05-01 at 13 51 39

**Comment by Jasstkn** [comment]
Screenshot 2020-05-01 at 13 57 26

**Comment by Jasstkn** [comment]
@ksatchit could you review this PR too, when you have time. Thanks!

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,3 +22,18 @@ jobs:
...
             - GITHUB_PAGES_REPO: litmuschaos/litmus-helm
           command: wget -O - https://raw.githubusercontent.com/litmuschaos/helm-github-pages/master/publish.sh | sh
+workflows:
+  version: 2
+  lint:
+    jobs:
+      - lint-charts
+  build:
+    triggers:
+      - schedule:
```
Comment (ksatchit) [inline .circleci/config.yml:32]: Could you explain this a bit @Jasstkn ?

**Comment by ksatchit** [comment]
Are the screenshots in comments from the build ? (I don't see those in the circleci log) @Jasstkn

**Comment by Jasstkn** [comment]
> Are the screenshots in comments from the build ? (I don't see those in the circleci log) @Jasstkn

Yep, I cleaned them.

File: .circleci/config.yml (hunk 1 above)
Comment (Jasstkn) [reply .circleci/config.yml:32]: Yep. lint-charts will be run at every commit, and lint-charts & build only on master every 23 and 53 minutes at every hour. I didn't find a more sophisticated solution. If you have one, share, please.

**Comment by ksatchit** [comment]
> > Are the screenshots in comments from the build ? (I don't see those in the circleci log) @Jasstkn
> 
> Yep, I cleaned them. You can find them here https://circleci.com/gh/Jasstkn/litmus-helm

Cool. Gives a good picture of what to expect when there are actual future errors. Thank you!

File: .circleci/config.yml (hunk 1 above)
Comment (ksatchit) [reply .circleci/config.yml:32]: Just thinking - wouldn't it be sufficient to run it on pull requests and commits to master over running them per schedule? We could use something similar to: https://github.com/litmuschaos/chaos-operator/blob/master/.circleci/config.yml#L109 - where run only lint-chart for PRs, lint-chart + build upon push/commit/merge.

Also, this is a good opportunity to fix/ensure that we do these workflows for master branch alone (currently the build runs for gh-pages branch as well and fails - if you notice)

Comment (Jasstkn) [reply .circleci/config.yml:32]: I will try, thanks for the proposal. It was my first time working with CircleCI, so that's why I needed a review.

**Comment by Jasstkn** [comment]
@ksatchit add filters option to build flow: it will first lint-charts again, then build.

File: .circleci/config.yml (hunk 1 above)
Comment (ksatchit) [reply .circleci/config.yml]: Thank you, looks great!

**Comment by ksatchit** [comment]
Thanks @Jasstkn for the awesome PR!

**Pull request merged**